output.go

```diff
@@ -35,21 +26,3 @@ type AggregatingOutput interface {
 	// Reset signals the the aggregator period is completed.
 	Reset()
 }
-
-type ServiceOutput interface {
-	// Connect to the Output
-	Connect() error
-	// Close any connections to the Output
-	Close() error
-	// Description returns a one-sentence description on the Output
-	Description() string
-	// SampleConfig returns the default configuration of the Output
-	SampleConfig() string
-	// Write takes in group of points to be written to the Output
-	Write(metrics []Metric) error
-
-	// Start the "service" that will provide an Output
-	Start() error
```

Comment:
Do service plugins replace `Start` and `Stop` implementations with `Connect` and `Close`?